# requested command
HybridImpedanceCommandGainsMsg command
---
# current command
HybridImpedanceCommandGainsMsg command